canonical-web-and-design/ubuntu.com
# Takeovers from Discourse

## Done

- Takeovers to Discourse

## QA

- Go to the demo homepage and check that the takeover is appearing.
- Go to the demo `/takeovers` and check that the rest of the takeovers are appearing
- Go to the [index table on Discourse](https://discourse.ubuntu.com/t/about-the-engage-pages-category/17229/2) and check that the data is consistent with this content (only active=true takovers are showing)


### Additional checks
- Change the index topic (line 279) of `app.py` to 18033 to check that the Spanish takeover is showing (you'll have to change your browser language to Spanish to test this). This data comes from the test [takeover data](https://discourse.ubuntu.com/t/test-engage-pages/18033), and it has a parameter active = true. On production we don't show this takeover, this is merely for testing.
- You can also break the markdown and see what happens. The expected result is that a default takeover of Ubuntu 20.10 is displayed instead.
- You may also want to play around with the `lang-skip` column to make sure e.g. first takeover does not show for italian.

## Issue / Card

Fixes #8406
